dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/NodeStats.java

```diff
@@ -0,0 +1,96 @@
+package com.dotcms.content.elasticsearch.business;
+
+import org.elasticsearch.common.unit.ByteSizeValue;
+
+public class NodeStats {
+    private final String name;
+    private final String host;
+    private int size;
+    private String prettySize;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'prettySize' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)